CHANGELOG

```diff
@@ -1,6 +1,10 @@
 devel
 -----
 
+* Fixed issue BTS-539 "Unsynchronized query kill while it's being finalized in
+  another thread was uncovered through `test-kill.js` of `communication_ssl` suite"
+  Fixed sync of query profiler  releasing with shutdown state changing in Query dtor
```

Comment:
The CHANGELOG entry is a little bit technical for the end user. How about something like _Fixed issue BTS-539: Possible (but unlikely) crash when killing an AQL query_?